packages/eth2.0-types/src/ssz/generators/validator.ts

```diff
@@ -15,10 +15,29 @@ export const ValidatorDuty = (ssz: IBeaconSSZTypes): SimpleContainerType => ({
   ],
 });
 
+export const CommitteeAssignment = (ssz: IBeaconSSZTypes, params: IBeaconParams): SimpleContainerType => ({
+  fields: [
+    ["validators", {
+      elementType: ssz.ValidatorIndex,
+      maxLength: params.MAX_VALIDATORS_PER_COMMITTEE,
+    }],
+    ["committeeIndex", ssz.CommitteeIndex],
+    ["slot", ssz.Slot],
+  ],
+});
+
 export const SyncingStatus = (ssz: IBeaconSSZTypes): SimpleContainerType => ({
   fields: [
     ["startingBlock", ssz.uint64],
     ["currentBlock", ssz.uint64],
     ["highestBlock", ssz.uint64],
   ],
 });
+
+export const AggregateAndProof = (ssz: IBeaconSSZTypes): SimpleContainerType => ({
```

Comment:
We are missing ssz spec test for this type, we can add that spec test in this PR or after we merge to 0.9.0. Just a note.

Reply:
I'll add it